crates/ruff_linter/src/checkers/ast/analyze/deferred_scopes.rs

```diff
@@ -48,6 +48,7 @@ pub(crate) fn deferred_scopes(checker: &Checker) {
         Rule::UnusedPrivateTypedDict,
         Rule::UnusedPrivateTypeVar,
         Rule::UnusedStaticMethodArgument,
+        Rule::UnusedUnpackedVariable,
```

Comment:
feedback/rant: took me forever to figure out that missing this line was why my tests weren't producing any lints. It makes sense in retrospect and I don't have a better idea for how to do it, but it was a bit frustrating.

Reply:
Sorry for that. Yeah, not sure what a good short-term fix for this is. My plan for Red Knot is to remove those manual checks entirely and instead have a more "formal" rule API that allows pre-computing in which phase a rule must run. This will hopefully help prevent such surprises